packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -329,6 +329,50 @@ module.exports = {
 
     const number = await entityManager.getNumberOfDraftRelations(id, model);
 
+    return {
+      data: number,
+    };
+  },
+  async getMultipleEntriesNumberOfDraftRelations(ctx) {
+    const { userAbility } = ctx.state;
+    const { model, ids } = ctx.params;
+
+    const entityManager = getService('entity-manager');
+    const permissionChecker = getService('permission-checker').create({ userAbility, model });
+
+    if (permissionChecker.cannot.read()) {
+      return ctx.forbidden();
+    }
+
+    const permissionQuery = await permissionChecker.sanitizedQuery.read(ctx.query);
+    const populate = await getService('populate-builder')(model)
+      .populateFromQuery(permissionQuery)
+      .build();
+
+    const idsArray = JSON.parse(ids);
+
+    if (idsArray.length <= 0) {
+      return {
+        data: 0,
+      };
+    }
+
+    let number = 0;
+
+    for (const id of idsArray) {
+      const entity = await entityManager.findOne(id, model, { populate });
```

Comment:
why not using a findMany here? and getting all the entities at once , that might increase the request performance

Reply:
Do you mean create a findMany inside the entityManager and inside it loop all the ids and call for every single id the findOne of the entityService?

Reply:
I was referring more to use `entityManager.find` , that would save you from loping througght the idsArray and getting all entities at once.

Reply:
ah ok great, thanks